core/src/apps/binance/layout.py

```diff
@@ -34,21 +34,18 @@ def make_input_output_pages(msg: BinanceInputOutput, direction: str) -> None:
     for txoutput in msg.outputs:
         make_input_output_pages(txoutput, "Confirm output")
 
-    await _confirm_transfer(items)
+    await _confirm_transfer(items, chunkify=bool(msg.chunkify))
 
 
-async def _confirm_transfer(inputs_outputs: Sequence[tuple[str, str, str]]) -> None:
+async def _confirm_transfer(
+    inputs_outputs: Sequence[tuple[str, str, str]], chunkify: bool = False
```

Comment:
remove default value

Reply:
Done in [7e03759](https://github.com/trezor/trezor-firmware/pull/3237/commits/7e03759ea7c3f0e2e56218ee1bd5cb18bef9c134)